hyperspy/samfire_utils/samfire_worker.py

```diff
@@ -27,8 +29,6 @@
 
 matplotlib.rcParams['backend'] = 'Agg'
```

Comment:
If the matplotlib backend is set before the import of hyperspy (and pyplot), I guess that there should be a reason. It not setting the matplotlib backend to `agg` should be removed.